apps/test/unit/code-studio/pd/regional_partner_mini_contact/RegionalPartnerMiniContactTest.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {expect} from '../../../../util/configuredChai';
```

Comment:
@jmkulwik For this new React component test, should Brendan be using [reconfiguredChai.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/util/reconfiguredChai.js)?

Reply:
Yep! I'll shoot a message out in #developers later today to let everybody know. We haven't sent any communication on this yet.